pytest_reportportal/listener.py

```diff
@@ -53,67 +64,83 @@ def pytest_runtest_makereport(self, item):
                 loglevel='ERROR',
             )
 
+        # Defining test result
         if report.when == 'setup':
             self.result = None
             self.issue = {}
-            if report.failed:
-                # This happens for example when a fixture fails to run
-                # causing the test to error
+
+        if report.failed:
```

Comment:
I don't think the below logic is incorrect compared to the previous implementation. Why did you combine pieces of logical applicable to reporting of setup fixtures and test items?

Reply:
We just applying or keeping the most bad result (pass -> skip -> fail) and phase doesn't mater.
Not sure where you see issue. Could you describe case